docs/SOPs/update_ena_runs_SOP.md

```diff
@@ -62,16 +64,27 @@ This is the SOP for fixing datasets in the issue: ebi-ait/hca-ebi-wrangler-centr
     python clear_run_accession_from_files.py <submission-uuid>
     ```
 
-4. Download all files from Ingest / Terra upload area to any directory inside `/nfs/production/hca/` in the EBI cluster.
+### 3 - Submit new sequencing runs
+1. Make sure that the metadata in Ingest contains sequencing experiment accessions. The submitter script will raise an error if
+   any of the assay processes has no accession. The assay processes in the submission should have the following
+   property:
+
+    ```
+    "insdc_experiment": {
+      "insdc_experiment_accession": "ERX4319109"
+    }
+    ```
+
+2. Download all files from Ingest / Terra upload area to any directory inside `/nfs/production/hca/` in the EBI cluster.
    [gsutil](https://cloud.google.com/storage/docs/downloading-objects) can be used for downloading the files
    The files may also be in the hca-util upload area but we should make sure they're valid. Using Ingest/Terra upload area means the files have already been validated before.
    Please prefer downloading the Terra upload area as downloading from Ingest upload area will incur cost to our AWS account.
 
-5. Checksum all the files.
+3. Checksum all the files.
     ``` 
     gsutil hash -hm gs://broad-dsp-monster-hca-prod-ebi-storage/prod/<project_uuid>/data/* | grep -A1 "hex" | awk -F"/" '{printf $4 $1}' | awk -F"--" '{for (i=1;i<NF;i++)print $i}' | awk -F":" '{print $1 $3}' > <md-filename>.txt
     ```
-6. Upload the files to Webin FTP upload area (could be in parallel with checksumming)
+4. Upload the files to Webin FTP upload area (could be in parallel with checksumming)
    ```
    $ cd <directory where you downloaded the files>
    $ lftp webin2.ebi.ac.uk -u <webin-user>
```

Comment:
I am missing a bit on how to get the webin user

Reply:
(the instructions may not necessarily tell you, but it should point you out to where/who to contact)

Reply:
Good point! I've got it from Karoly before, it's the DSP webin user 😅